app/models/solidus_paypal_braintree/configuration.rb

```diff
@@ -8,7 +8,7 @@ class Configuration < ::Spree::Base
       label: { availables: %w[checkout credit pay buynow paypal installment], default: 'checkout' },
       tagline: { availables: %w[true false], default: 'false' },
       layout: { availables: %w[horizontal vertical], default: 'horizontal' },
-      messaging: { availables: %w[true false], default: 'true' }
+      messaging: { availables: %w[true false], default: 'false' }
```

Comment:
If it's disabled by default, should we explain in the readme how to enable messaging?

Reply:
I added this to the `paypal financing messaging` section:
```
To enable this option, you'll need to use the `checkout` flow, and set the `paypal button messaging` option to `true` in your Braintree configuration.
```
Does that work?

Reply:
Oh right, sorry!